Formula/python-certifi.rb

```diff
@@ -0,0 +1,36 @@
+class PythonCertifi < Formula
+  desc "Mozilla CA bundle for Python"
+  homepage "https://github.com/certifi/python-certifi"
+  url "https://files.pythonhosted.org/packages/98/98/c2ff18671db109c9f10ed27f5ef610ae05b73bd876664139cf95bd1429aa/certifi-2023.7.22.tar.gz"
+  sha256 "539cc1d13202e33ca466e88b2807e29f4c13049d6d87031a3c110744495cb082"
+  license "MPL-2.0"
+
+  depends_on "python@3.10" => [:build, :test]
+  depends_on "python@3.11" => [:build, :test]
+  depends_on "python@3.9" => [:build, :test]
+  depends_on "ca-certificates"
+
+  def pythons
+    deps.map(&:to_formula).sort_by(&:version).filter { |f| f.name.start_with?("python@") }
+  end
+
+  def install
+    pythons.each do |python|
+      python_exe = python.opt_libexec/"bin/python"
+      system python_exe, *Language::Python.setup_install_args(prefix, python_exe)
+
+      # Use brewed ca-certificates PEM file instead of the bundled copy
+      site_packages = Language::Python.site_packages("python#{python.version.major_minor}")
+      rm prefix/site_packages/"certifi/cacert.pem"
+      (prefix/site_packages/"certifi").install_symlink Formula["ca-certificates"].pkgetc/"cert.pem" => "cacert.pem"
+    end
+  end
+
+  test do
+    pythons.each do |python|
+      python_exe = python.opt_libexec/"bin/python"
+      output = shell_output("#{python_exe} -c 'import certifi; print(certifi.where())'").chomp
```

Comment:
```suggestion
      output = shell_output("#{python_exe} -m certifi").chomp
```
I think this is equivalent